lib/draw.js

```diff
@@ -164,6 +164,54 @@ export default function (self) {
     self.ctx.fill();
     return ml + aw + mr;
   }
+  function drawCellTreeIcon(cell, tree, rowTree) {
+    const pc = rowTree ? tree.parentCount : 0;
```

Comment:
Can we write out variable names like `pc/rc/mt/mr/ml` etc to something more descriptive?

No hard rule that _all_ need to be written out (I mean, stuff like `x` and `y` are obvious, as are counters like `i`), but while it's a little more annoying to write the code, writing the more semantic variables out makes it _a lot_ easier to read and debug afterwards